### PR TITLE
Adaptive cards support

### PR DESCRIPTION
**Pull request by asantamarina**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py [hunk 1]
```diff
@@ -613,6 +613,49 @@ def process_ask_user(message: str) -> dict:
     return create_adaptive_card(body, actions)
 
 
+def add_data_to_actions(card_json, data_value):
```
Comment (rshunim) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py:616]: Add here typing for the function's arguments and for the return value too

**Review by rshunim — COMMENTED** [review]
Hi @asantamarina, thanks for your contribution!
Start reviewing the PR

**Comment by rshunim** [comment]
Hi @asantamarina, @merit-maita and me want to set a demo with you

**Comment by thefrieddan1** [comment]
Hi @asantamarina 
Lets please schedule a session to demonstrate the functionality of this contribution.
Thanks

File: Packs/MicrosoftTeams/ReleaseNotes/1_5_1.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
+#### Integrations
+
+##### Microsoft Teams
+
+- Adding support for Adaptive Cards.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/ReleaseNotes/1_5_1.md:6]:
```suggestion
Added support for Adaptive Cards. 
```

File: Packs/MicrosoftTeams/ReleaseNotes/1_5_1.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+##### Microsoft Teams
+
+- Adding support for Adaptive Cards. 
+
+#### Scripts
+
+##### MicrosoftTeamsAsk
+
+- Adding support for Adaptive Cards.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/ReleaseNotes/1_5_1.md:12]:
```suggestion
- Added support for Adaptive Cards.
```

**Comment by ShirleyDenkberg** [comment]
@rshunim @thefrieddan1 @merit-maita Doc review completed.

**Pull request merged**